app/services/notifications/new_follower/send.rb

```diff
@@ -7,6 +7,9 @@ class Send
       #   * :followable_type [String] - "User" or "Organization"
       #   * :follower_id [Integer] - user id
       def initialize(follow_data, is_read = false)
+        # we explicitly symbolize_keys because FollowData.new will fail otherwise with an error of
```

Comment:
good call! this is because of how delayed job is different from sidekiq.

I wonder if we should add a spec in the services spec to make sure this doesn't regress

Reply:
I really like this idea! I gave it a shot. Let me know if you had something else in mind - I'm happy to make changes.